### PR TITLE
IOS-2444 Membership | Add ownership info sheet

### PR DESCRIPTION
**Pull request by ignatovv**
image

### CODE REVIEW

File: Anytype/Sources/PresentationLayer/Flows/MembersipCoordinator/MembershipCoordinatorModel.swift [hunk 1]
```diff
@@ -8,20 +8,32 @@ final class MembershipCoordinatorModel: ObservableObject {
...
         tierSelectionAssembly: MembershipTierSelectionAssemblyProtocol,
         emailVerificationAssembly: EmailVerificationAssemblyProtocol
     ) {
+        self.membershipService = membershipService
         self.membershipAssembly = membershipAssembly
         self.tierSelectionAssembly = tierSelectionAssembly
         self.emailVerificationAssembly = emailVerificationAssembly
     }
     
+    func onAppear() {
```
Comment (joe-pusya) [inline Anytype/Sources/PresentationLayer/Flows/MembersipCoordinator/MembershipCoordinatorModel.swift:31]: I think you can just mark onAppear as async here

File: Anytype/Sources/PresentationLayer/Modules/Membership/TierSelection/MembershipTierSelectionViewModel.swift [hunk 2]
```diff
@@ -4,20 +4,29 @@ import Services
...
         membershipService: MembershipServiceProtocol,
         showEmailVerification: @escaping (EmailVerificationData) -> ()
     ) {
-        self.tier = tier
+        self.tierToDisplay = tierToDisplay
         self.membershipService = membershipService
         self.showEmailVerification = showEmailVerification
     }
     
+    func onAppear() {
```
Comment (joe-pusya) [inline Anytype/Sources/PresentationLayer/Modules/Membership/TierSelection/MembershipTierSelectionViewModel.swift:24]: and here)

File: Anytype/Sources/PresentationLayer/Flows/MembersipCoordinator/MembershipCoordinatorModel.swift (hunk 1 above)
Comment (ignatovv) [reply Anytype/Sources/PresentationLayer/Flows/MembersipCoordinator/MembershipCoordinatorModel.swift:31]: membershipService.getStatus() throws
but View modificator .task {} is non throwable
So I have to do catch in view body or wrap call in Task here

Comment (joe-pusya) [reply Anytype/Sources/PresentationLayer/Flows/MembersipCoordinator/MembershipCoordinatorModel.swift:31]: got it!

**Pull request merged**